dll/cpl/appwiz/addons.c

```diff
@@ -362,9 +362,14 @@ static DWORD WINAPI download_proc(PVOID arg)
 
     hres = URLDownloadToFileW(NULL, GeckoUrl, tmp_file, 0, &InstallCallback);
     if(FAILED(hres)) {
-        ERR("URLDownloadToFile failed: %08x\n", hres);
         if (LoadStringW(hApplet, IDS_DWL_FAILED, message, sizeof(message) / sizeof(WCHAR))) {
```

Comment:
There is no need to even load the string in the `E_ABORT` case,
but either way is fine with me.